src/common/transformers/SUFFIX.js

```diff
@@ -1,34 +1,10 @@
-export const suffix = (value, str) => {
-    if (!value) {
-        return str;
-    }
+import { transformerWithArg } from './transformer';
 
-    if (typeof value === 'string') {
-        return value.concat(str);
-    }
+export const suffix = (value, str) =>
+    typeof value === 'string' ? value.concat(str) : value;
```

Comment:
Same problem as with SUFFIX: no more applicable on an array.
We have to update [the documentation](https://www.gitbook.com/book/lodex/lodex-user-documentation/edit#/edit/master/Administration/Modèle/Transformers/SUFFIX.md).